scripts/toolkit/modules/developers/auth/integration_runner.Group.yaml

```diff
@@ -0,0 +1,332 @@
+name: developers
+members:
+  - '123456789012'
+metadata:
+  origin: "cognite-toolkit"
+capabilities:
+  - projectsAcl:
+      actions:
+        - LIST
+        - READ
+        - UPDATE
+      scope:
+        all: {}
+  - analyticsAcl:
+      actions:
+      - READ
+      - EXECUTE
+      - LIST
+      scope:
+        all: {}
+  - annotationsAcl:
+      actions:
+      - WRITE
+      - REVIEW
+      - READ
+      - SUGGEST
+      scope:
+        all: {}
+  - assetsAcl:
+      actions:
+      - WRITE
+      - READ
+      scope:
+        all: {}
+  - auditlogAcl:
+      actions:
+      - READ
+      scope:
+        all: {}
+  - dataModelInstancesAcl:
+      actions:
+      - WRITE_PROPERTIES
+      - WRITE
+      - READ
+      scope:
+        all: {}
+  - dataModelsAcl:
+      actions:
+      - WRITE
+      - READ
+      scope:
+        all: {}
+  - datasetsAcl:
+      actions:
+      - WRITE
+      - OWNER
+      - READ
+      scope:
+        all: {}
+  - diagramParsingAcl:
+      actions:
+        - WRITE
+        - READ
+      scope:
+        all: { }
+  - digitalTwinAcl:
+      actions:
+      - WRITE
+      - READ
+      scope:
+        all: {}
+  - documentFeedbackAcl:
+      actions:
+      - CREATE
+      - READ
+      - DELETE
+      scope:
+        all: {}
+  - documentPipelinesAcl:
+      actions:
+      - WRITE
+      - READ
+      scope:
+        all: {}
+  - entitymatchingAcl:
+      actions:
+      - WRITE
+      - READ
+      scope:
+        all: {}
+  - eventsAcl:
+      actions:
+      - WRITE
+      - READ
+      scope:
+        all: {}
+  - extractionConfigsAcl:
+      actions:
+      - WRITE
+      - READ
+      scope:
+        all: {}
+  - extractionPipelinesAcl:
+      actions:
+      - WRITE
+      - READ
+      scope:
+        all: {}
+  - extractionRunsAcl:
+      actions:
+      - WRITE
+      - READ
+      scope:
+        all: {}
+  - filePipelinesAcl:
+      actions:
+      - WRITE
+      - READ
+      scope:
+        all: {}
```

Comment:
What's a file pipeline?

Reply:
don't know, this is just a group with all possible capabilities.